webrecorder/wacz-uploader
# Initial UI web component setup

- Adds web dev server to serve files and watch changes to bundle with `npm run dev`
- Initial layout updates to `index.html` using a [Shoelace card component](https://shoelace.style/components/card)

### Manual testing
1. Install dependencies `npm i`
2. Run `npm run dev`. Verify app opens in browser

### Screenshots
<img width="1041" alt="Screen Shot 2022-10-18 at 4 19 33 PM" src="https://user-images.githubusercontent.com/4672952/196562804-b3c2226a-f7d2-43df-a771-1b5dff9f76c1.png">
<img width="447" alt="Screen Shot 2022-10-18 at 4 19 42 PM" src="https://user-images.githubusercontent.com/4672952/196562808-fe62ed9c-38c6-481a-b476-9393d82148a2.png">
